README.md

```diff
@@ -7,3 +7,11 @@ The aim of the Github SPA is to present actions and issues that need to be surfa
 PRs and Issues are displayed in order of most recently submitted.  For PRs, the application displays the title, the date created, submitted by, and if merge conflicts exist.  For Issues, the issue number, issue title, to whom the issue was assigned, and any labels associated with the issue.  For Milestones, the application displays the title of the milestone, the due date if it exists, and what percentage of the milestone is complete.  
 
 On each view there is the option to step back to the previous view or to logout and login under a different account.  
+
+<img width="358" alt="screen shot 2017-09-25 at 2 41 53 pm" src="https://user-images.githubusercontent.com/12532173/30833027-2c5e0c86-a202-11e7-89c4-b8e776b81bc1.png">
```

Comment:
These are great @SSabu, thanks! Could you please add some [horizontal rules](https://daringfireball.net/projects/markdown/syntax#hr) between the images?